### ISSUE TITLE
Too many settings to use the mobile function.

### ISSUE THREAD

**Issue by rlaehddnrwiki** [flutter/flutter#51614]
## Use case
Too much configuration is required to use the Google service api.

## Proposal
Don't distinguish between web and mobile apps, even if you have only a lib folder, you can use all the features natively as a multi-platform and remove the android or ios folder. 
Make abstraction.
That's what developer want

**Comment by goderbauer** [comment]
It's unclear to me what this issue is suggesting or talking about. Can you rephrase it?

**Comment by VladyslavBondarenko** [comment]
@rlaehddnrwiki 
Without additional information, I'm going to close that proposal for now. 
Provide more details and I'll reopen it
